code/datums/keybinding/human.dm

```diff
@@ -56,6 +56,9 @@
 	if(!equipped_belt.contents.len) // nothing to take out
 		to_chat(user, "<span class='notice'>There's nothing in your belt to take out.</span>")
 		return TRUE
+	var/datum/component/storage/STR = equipped_belt.GetComponent(/datum/component/storage)
+	if(!STR.can_be_opened)
+		STR.quick_empty(H)
```

Comment:
This seems like it could be somewhat inconsistent and mess with people's muscle memory if they actually use the hotkey. Perhaps it should just fail instead and do nothing.

Reply:
God damn, I forgot about this.